blazor/treemap/accessibility.md

```diff
@@ -39,7 +38,7 @@ The accessibility compliance for the TreeMap component is outlined below.
 
 ## WAI-ARIA attributes
 
-The TreeMap component follows the [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) patterns to meet the accessibility. The following ARIA attributes are used in the TreeMap component:
+The Blazor TreeMap component follows the [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) patterns to meet the accessibility. The following ARIA attributes are used in the TreeMap component:
```

Comment:
"WAI-ARIA" in second line is missing. The link or code style not provided.

Reply:
Hi @sabari-senthamaraikannan ,

I modified the reference link.

Regards,
@Backiaraj